packages/glimmer-runtime/lib/builder.ts

```diff
@@ -279,6 +279,14 @@ export class ElementStack {
     return fragmentBounds;
   }
 
+  insertTextBefore(nextSibling: Node, text: string): Fragment {
```

Comment:
I'm not sure `appendInsertion` makes sense because it implies that we're inserting at the end always which is not the case.